.github/workflows/test.yml

```
@@ -144,10 +144,10 @@ jobs:
           etc/bin/shellchecker
           yarn run lint
 
-      # - name: MyPy
-      #   working-directory: rose
-      #   run: |
-      #     mypy --explicit-package-bases
+      - name: MyPy
+        working-directory: rose
+        run: |
+          mypy --explicit-package-bases
```

Don't need this anymore, has been added to `mypy.ini`
```suggestion
          mypy
```